test/semanticValidatorTests.ts

```diff
@@ -100,6 +100,16 @@ describe("Semantic validation", () => {
         ".paths['/providers/Microsoft.ErrorPath/{name}'].get.parameters[0].required"
       );
     });
+    // path brackets
+    it("should fail when path brackets are deployed as a single", async () => {
+      const specPath = `${testPath}/semanticValidation/specification/validateSwaggerSchema/internalErrors/pathBrackets.json`;
+      const result = await validate.validateSpec(specPath, undefined);
+      assert(result.validityStatus === false);
+      assert.strictEqual(
+        result.validateSpec?.errors?.[0].code,
+        constants.ErrorCodes.InternalError.name
```

Comment:
Can we turn this error to a new error code instead of internal error as it's swagger error and user has to fix it?